### ISSUE TITLE
UCE prevents My Account page from adding additional payment methods for other gateways

### ISSUE THREAD

**Issue by johndcoy** [woocommerce/woocommerce-gateway-stripe#2631]
**Describe the bug**
When trying to add an additional payment method for [WooCommerce Square](https://woocommerce.com/products/square/) from the My Account page, the following message appears:

`Your card number is incomplete.`

![Screenshot](https://d.pr/i/HDQOk0+)
Screenshot link: https://d.pr/i/HDQOk0

It works as expected with UCE disabled.

**To Reproduce**
1. Enable UCE for Stripe
2. Enable Square for WooCommerce
3. Attempt to add a payment method to Square from **My Account > Payment Methods**

**Expected behavior**
Stripe should be ignored when fields are collapsed.

**Screenshots**

![Screenshot](https://d.pr/i/WcZgaI+)
Screenshot link: https://d.pr/i/WcZgaI

**Comment by ryanr14** [comment]
I've run into this same issue between Square & Stripe, but I see a slightly different front end error.

> An error occurred, please try again or try an alternate form of payment.

<img width="598" alt="Screenshot 2024-06-12 at 10 02 12 PM" src="https://github.com/woocommerce/woocommerce-gateway-stripe/assets/867748/c6534b52-2ac2-4a84-8763-bae5ccd4ffcc">


With both plugins activated, I get the following error in the logs. Once Stripe is deactivated, I can save payment methods again in the My Account area.

```
2024-06-13T01:58:21+00:00 Notice 3D Secure Verification Token is missing  
Additional context
{
    "_legacy": true
}
```

8297209-zen

**Comment by Mayisha** [comment]
This issue is fixed in `8.4.0`. 
It was mainly fixed to resolve a similar conflict with WooPayment ([PR](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3049))

The merchant in 8297209-zd-a8c has version `8.3.1` of the Stripe plugin. With Stripe `8.3.1` and Square, I can reproduce the issue. With Stripe `8.4.0` and Square, this error no longer happens.

**Comment by Mayisha** [comment]
I am closing this issue as it was fixed in version 8.4.0, and we did not receive any further complaints.
